### DISCUSSION TITLE
I need help getting it to work

### DISCUSSION THREAD

**Discussion by sakgoyal** [searxng/searxng#1542] [Q&A]
I am trying to set up my own instance of searxng. 
I am following NetowrkChuck's guide on how he made his but my instance doesnt seem to be accessible. 

I am hosting my instance on an oracle VCN. I have a public ip as well. 
I installed docker/docker compose, I changed the hostname to the public ip of the oracle vm, and docker shows 3 containers up and running. but whenever I try to access the instance by going to the public ip of the vm, I keep getting a connection timed out. 
I have dont nothing else other than follow networkchuck's video. so if theres a missing step, please lmk

I dont know what is happening.

**Comment by return42** [comment]
> I am following NetowrkChuck's guide

Did you asked NetowrkChuck for help, what does he recommend?

**Reply by sakgoyal** [reply]
he has like 3M subscribers and his discord has like 100k people. i doubt he will respond specifically to me.

**Reply by return42** [reply]
OK, I understand .. please note that we (the SearXNG developers) do not have the time to verify installation guides from others. By this I can give you only some general hints ..

- If you don't get an answer here; in our Matrix [#searxng:matrix.org](https://matrix.to/#/#searxng:matrix.org) chat there is a community with a lot of experience (a number of the chat participant coming from NetowrkChuck's guide)

- If you think you have an issue with docker setup, may have a look at the (closed & open) issue at [searxng-docker](https://github.com/searxng/searxng-docker/issues?q=is%3Aissue)

- Several docker users reported problems and solution here in the https://github.com/searxng/searxng/issues/1397 thread

**Comment by whzy1314** [comment]
If you are using the docker to setup, in the .env file, try to use a actual fqdn instead of IP address as hostname. Caddy may not able to get you SSL cert with IP address as hostname

**Reply by sakgoyal** [reply]
I actually just bought a domain. I will be trying the normal way to set this up soon

**Reply by mrpaulblack** [reply]
Make sure that your servers public IP is actually accessible and not blocked by a firewall. I remember with oracle there are some firewall settings hidden deep in some menus...

You can check it by removing the SearXNG containers on the server with `docker-compose down` and then starting a nginx server with `docker run -it --rm -p 80:80 nginx:latest` no try to open up `http://[yourserverip]` if you can see the example site its a problem with SearXNG, if you still get a connection timeout you have misconfigured your server somehow in that case, so please check the oracle docs if it still times out :+1:

**Comment by chisss07** [comment]
I have a question, which also came from NetworkChuck. 
I got everything to run normally using the default settings. However, the second that I make a change to the settings.yml file, the server never wants to come back up again. Doesn't matter what it is that I change, I can put a space, remove it, save the file and the server is done. 
Any ideas?
